tools/cli/workflow_batch_commands.go

```diff
@@ -134,7 +136,7 @@ func DescribeBatchJob(c *cli.Context) error {
 func ListBatchJobs(c *cli.Context) error {
 	domain, err := getRequiredOption(c, FlagDomain)
 	if err != nil {
-		return commoncli.Problem("Required flag not found: ", err)
```

Comment:
please revert the change here and below. This is expected to return a problem error.